src/qr.cpp

```diff
@@ -2,11 +2,10 @@
 
 neroshop::QR::QR() {}
 ////////////////////
-neroshop::QR::QR(std::string filename, int image_size, int min_module_pixel_size, std::string text,
+neroshop::QR::QR(std::string filename, int image_size, std::string text,
```

Comment:
Is there a reason to make a copy of the input argument strings, `filename` and `text`? If not, I would avoid them with
```cpp
neroshop::QR::QR( const std::string& filename, int image_size, const std::string& text, bool overwrite, qrcodegen::QrCode::Ecc ecc)
```

Reply:
You're right, I made the modification on the new PR.